swiss_locator/core/filters/swiss_locator_filter.py

```diff
@@ -427,6 +431,79 @@ def triggerResult(self, result: QgsLocatorResult):
             if self.settings.value("show_map_tip"):
                 self.show_map_tip(swiss_result.layer, swiss_result.feature_id, point)
 
+        # Vector tiles
+        elif type(swiss_result) == VectorTilesLayerResult:
+            params = dict()
+            params["styleUrl"] = swiss_result.style or ""
+            params["url"] = swiss_result.url
+            params["type"] = "xyz"
+            params["zmax"] = "14"
```

Comment:
is this the correct max zoom?

Reply:
We can find the min/max levels in a metadata file like this: 

https://vectortiles.geo.admin.ch/tiles/ch.swisstopo.base.vt/v1.0.0/tiles.json

For all Swiss Vector Tile layers they set the level 14 as maximum (level 14 goes up to buildings).

Should we always go read the min/max zoom to the corresponding online metadata JSON file?

Reply:
perfect, all good like this, maybe add a comment how to retrieve this?